aiozipkin/tracer.py

```diff
@@ -96,7 +96,10 @@ def create(zipkin_address: str,
 
     async def build_tracer() -> Tracer:
         sampler = Sampler(sample_rate=sample_rate)
-        transport = Transport(zipkin_address, send_interval=send_interval, loop=loop)
+        if not zipkin_address:
+            transport = cast(Transport, StubTransport())
```

Comment:
this shouldn't be needed anymore, I'll just make Trace accept TransportABC, but I'll try to add some tests to it first...

Reply:
Agreed cast should be removed. 

Lets think about API should we create new function something like:
```python
def create_with_transprot(
    transport: OptTransport: None,,
    local_endpoint: Endpoint,
    loop: OptLoop=None) -> Awaitable[Tracer]:
```
?

Reply:
Looks good, but maybe as a separate PR ?

Reply:
agreed